locales/genshin/de.json

```diff
@@ -71,151 +71,150 @@
-    "main_dps": "Main DPS",
-    "sub_dps": "Sub DPS",
-    "support": "Support"
+    "title": "Genshin Impact Rangliste (Beste Figuren)",
+    "description": "Die besten Figuren und ihre Builds nach Stärke, Überlebensfähigkeit und Vielseitigkeit sortiert, um Inhalte zu bewältigen.",
+    "tierlist": "Rangliste",
+    "main_dps": "Haupt-Kämpfer",
+    "sub_dps": "Zusatz-Kämpfer",
+    "support": "Unterstützung"
   },
   "tierlist_weapons": {
-    "title": "Genshin Impact Liste der Waffenstufen (Beste Waffen)",
-    "description": "Die besten Waffen sortiert nach Stärke, Lebensfähigkeit und Vielseitigkeit, um Inhalte zu löschen.",
-    "tierlist_weapons": "Beste Waffen",
-    "main_dps": "Main DPS",
-    "sub_dps": "Sub DPS",
-    "support": "Support"
+    "title": "Genshin Impact Rangliste der Waffen (Beste Waffen)",
+    "description": "Die besten Waffen nach Stärke, Überlebensfähigkeit und Vielseitigkeit sortiert, um Inhalte zu bewältigen.",
+    "tierlist_weapons": "Waffenrangliste",
+    "main_dps": "Haupt-Kämpfer",
+    "sub_dps": "Zusatz-Kämpfer",
+    "support": "Unterstützung"
   },
   "food": {
     "title": "Genshin Impact Liste der Kochrezepte",
-    "description": "Entdecken Sie alle Kochrezepte und das beste Essen für Ihr Team.",
-    "food": "Nahrung",
+    "description": "Entdecke alle Kochrezepte und das beste Essen für deine Truppe.",
+    "food": "Gerichte",
     "all": "Alle",
     "normal": "Normal",
     "delicious": "Lecker",
-    "suspicious": "Misstrauisch",
-    "special": "Speziell",
-    "search": "Suche..."
+    "suspicious": "Misslungen",
+    "special": "Besonders",
+    "search": "Suchen..."
   },
   "ingredients": {
-    "title": "Genshin Impact Zutatenliste für das Kochen",
-    "description": "Entdecken Sie alle Kochzutaten.",
-    "cooking_ingredient": "Zutat",
+    "title": "Genshin Impact Liste der Kochzutaten",
+    "description": "Entdecke alle Kochzutaten.",
+    "cooking_ingredient": "Kochzutat",
     "name": "Name"
   },
   "potions": {
-    "title": "Genshin Impact Tränkliste",
-    "description": "Entdecken Sie alle Alchemie-Rezepte und die besten Tränke und Öle für Ihr Team.",
+    "title": "Genshin Impact Liste der Tränke",
+    "description": "Entdecke alle Alchemierezepte und die besten Tränke und Öle für deine Truppe.",
     "potions": "Tränke",
     "name": "Name",
     "rarity": "Seltenheit",
-    "effect": "Effect"
+    "effect": "Effekt"
   },
   "teams": {
-    "title": "Genshin Impact Beste Team-Comp | Anleitung zum Aufbau von Partys",
-    "description": "Dies ist eine Anleitung, um die beste Party in Genshin Impact zu machen. Erfahren Sie, wie Sie die beste Party machen! Wir stellen die beste Gruppenzusammensetzung für jede Aufgabe vor, darunter das Erkunden von Gebieten, das Töten von Feldbossen und mehr!",
-    "teams": "Team-Comp",
-    "best_team_comp": "Beste Teamzusammensetzung",
-    "character_team": "Bestes Team für {name}",
-    "substitute_characters": "Substitute Characters",
```

Comment:
I removed this line because it seems not in use anymore. It is also missing in the english file.